### ISSUE TITLE
Responsive navbar

### ISSUE THREAD

**Issue by tobiasBora** [uikit/uikit#4315]
### Summary

While bootstrap has responsive navbar with a nice Hamburger menu appearing on small devices, uikit does not have such feature.  See for example [this post](https://stackoverflow.com/questions/44898526/responsive-navbar-in-uikit-3), where most solutions are nasty.

### Priority

4: to me the main part for a website is maybe the navigation menu. Since it's super easy to do with other frameworks, it's really a problem not to have proper solutions in uikit.

**Comment by flyboarder** [comment]
I use a single responsive navbar with hamburger menu on desktop and mobile. 

The question on SO is from 3+ years ago, I believe this is no longer relevant.

**Comment by tobiasBora** [comment]
Could you give me an example? Can't find anything in the doc.

**Comment by flyboarder** [comment]
provided the page is setup to be responsive, ie. has a [meta tag](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag) then the entire site should be responsive.

**Comment by tobiasBora** [comment]
Obviously, this is not enough to have a hamburger on mobile screens (see code at the end):

![image](https://user-images.githubusercontent.com/2164118/99504351-ba699880-297f-11eb-931b-ee059304ec37.png)

It does not even provide a way to scroll only the bar: the whole page extends to the right, making ugly borders on the right of the page, and the background color is not even displayed on the extended menu:

![image](https://user-images.githubusercontent.com/2164118/99504538-f4d33580-297f-11eb-8836-40ac845586f3.png)

I would, a minima, expect a scrollable bar that does not add a margin on the whole page, for example like what's done on Bootstrap main page:

![image](https://user-images.githubusercontent.com/2164118/99504768-4380cf80-2980-11eb-9b88-64a16c261c8f.png)

And my initial request was to display on smaller screen instead this kind of hamburger menu:

![image](https://user-images.githubusercontent.com/2164118/99505036-99ee0e00-2980-11eb-91c3-db7c736f60d4.png)

If I missed an essential class to add, please let me know.

MWE:
```
<!doctype html>
<html class="no-js" lang="">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <title>Test</title>
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- UIkit CSS -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/uikit@3.5.9/dist/css/uikit.min.css" />

    <!-- UIkit JS -->
    <script src="https://cdn.jsdelivr.net/npm/uikit@3.5.9/dist/js/uikit.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/uikit@3.5.9/dist/js/uikit-icons.min.js"></script>
  </head>
  <body>
    <nav class="uk-navbar-container" uk-navbar>
      <div class="uk-navbar-left">
        <ul class="uk-navbar-nav">
          <li class="uk-active"><a href="#">Hello</a></li>
          <li class="uk-parent"><a href="#">This</a></li>
          <li class="uk-parent"><a href="#">Is</a></li>
          <li class="uk-parent"><a href="#">A Very</a></li>
          <li class="uk-parent"><a href="#">Long</a></li>
          <li class="uk-parent"><a href="#">Navigation</a></li>
          <li class="uk-parent"><a href="#">Menu</a></li>
          <li class="uk-parent"><a href="#">Bar</a></li>
        </ul>
      </div>
    </nav>
  </body>
</html>
```

**Comment by flyboarder** [comment]
You will need to include a hamburger icon and menu yourself, then toggle visibility on mobile screen sizes.
